drivers/tmp006/include/tmp006.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2014 PHYTEC Messtechnik GmbH
+ *               2017 HAW Hamburg
```

Comment:
I remember this one and as I said, I think it's a good idea in general to allow the use of external drivers implementation. How will this work with pseudomodules ? Some generic drivers use them to specify a precise device, see bmx280 for instance.
Anyway, for the moment, I think we should stick to the current design and move the discussion in #6923